cmake/NeighborhoodTablesConfig.cmake

```diff
@@ -0,0 +1,37 @@
+#------------------------------------------------------------------------------
+# DGtal Configuration file for LookUpTables
+#------------------------------------------------------------------------------
+
+# ------ AT CONFIGURE TIME ------ #
+#--- Configuration of the src/topology/tables/NeighborhoodTables.h.in
+set(TABLE_DIR ${PROJECT_BINARY_DIR}/src/DGtal/topology/tables)
+configure_file(
+  ${PROJECT_SOURCE_DIR}/src/DGtal/topology/tables/NeighborhoodTables.h.in
+  ${PROJECT_BINARY_DIR}/src/DGtal/topology/tables/NeighborhoodTables.h)
+
+#--- Unpack files at cmake configuration time.
+file(GLOB DGTAL_TABLES_COMPRESSED
+     ${PROJECT_SOURCE_DIR}/src/DGtal/topology/tables/*.tar.gz)
+set(unzip_folder ${PROJECT_BINARY_DIR}/src/DGtal/topology/tables/)
+message(STATUS "Decompressing look up tables to: ${unzip_folder}")
+foreach(zip_table ${DGTAL_TABLES_COMPRESSED})
+  execute_process(
+    COMMAND           ${CMAKE_COMMAND} -E tar xzf ${zip_table}
+    WORKING_DIRECTORY ${unzip_folder}
```

Comment:
[CMake](https://cmake.org/cmake/help/v3.2/manual/cmake.1.html) provides it, and it is mulit-platform, but I am not really sure what version of cmake is required.